src/sass/static/pages.scss

```diff
@@ -1624,6 +1624,35 @@ body #not_authenticated_financial ul.checked > li {
     }
 }
 
+#msg_main.close-account {
```

Comment:
it would be better to have a more unique identifier that's specific to the element rather than two generic identifiers grouped together

Reply:
The modal has now a unique class.